litmus-portal/frontend/src/views/Settings/AccountsTab/PersonalDetails/index.tsx

```diff
@@ -145,7 +148,9 @@ const PersonalDetails: React.FC = () => {
                   <Loader size={20} />
                 </div>
               ) : (
-                <>Save Changes</>
+                <>
+                  {t('settings.accountsTab.personalDetails.button.saveChanges')}
+                </>
```

Comment:
All these would require changing if children are removed from props, this would anyway be removed with the new button component from Kubera-ui